frontend/src/modules/Home/Components/Home.tsx

```diff
@@ -1,79 +1,56 @@
-import React from 'react'
 import {
   StyledBackground,
   StyledContainer,
-  StyledHeaderText,
   StyledLeftPanel,
   StyledRightPanel,
-  StyledWelcomeText,
   StyledWhiteActionText,
 } from 'Home/Styles/Home.style'
-import Button from '@mui/material/Button'
 import { adminSignIn } from '@fire'
-// import { ReactComponent as Google } from '@assets/img/googleicon.svg'
-import { ReactComponent as Microsoft } from '@assets/img/microsofticon.svg'
-import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon'
-import { Box } from '@mui/material'
-import teacher from '@assets/img/teacher.svg'
-
-// function GoogleIcon(props: SvgIconProps) {
-//   return <SvgIcon inheritViewBox component={Google} {...props} />
-// }
-
-function MicrosoftIcon(props: SvgIconProps) {
-  return <SvgIcon inheritViewBox component={Microsoft} {...props} />
-}
+import { Box, Button, Typography } from '@mui/material'
+import matchimg from '@assets/img/matching.svg'
+import { Link } from 'react-router-dom'
 
 export const Home = () => {
   return (
     <StyledBackground>
       <StyledContainer>
         <StyledLeftPanel>
           <StyledWhiteActionText>
-            Connect students, create groups
+            <Typography variant="h2" fontWeight={'600'}>
+              Connect students. <br />
+              Create groups.
+            </Typography>
```

Comment:
I think you should consolidate instances where an element only has one child element, into just one element, especially when the components aren't used elsewhere. This collapsing of elements removes redundant HTML and puts all the styling on that element in one place.

For example,
```js
<Box sx={{ a: x }}>
  <Box sx={{ b: y }}>
    ...
  </Box>
</Box>
```
is cleaner as
```js
<Box sx={{ a: x, b: y }}>
  ...
</Box>
```